vividus-util/build.gradle

```diff
@@ -18,6 +21,9 @@ dependencies {
 
     implementation(group: 'com.fasterxml.jackson.datatype', name: 'jackson-datatype-jsr310')
 
+    implementation(group: 'com.google.zxing', name: 'core', version: zxingVersion)
+    implementation(group: 'com.google.zxing', name: 'javase', version: zxingVersion)
```

Comment:
QR code reader shouldn't be a part of utils: it's used in one place and brings extra dependencies, which may not be useful for other test projects (e.g. REST API testing) so this logic should be moved to `extension-selenium`